scm/src/gmtb_scm.F90

```diff
@@ -37,11 +37,11 @@ subroutine gmtb_scm_main_sub()
   character(len=16) :: logfile_name
 
   call get_config_nml(scm_state)
-
+  
```

Comment:
These extra whitespaces are not required, better to remove (in total four places in this file).

Reply:
addressed in last commit